### PR TITLE
add rate button

### PR DESCRIPTION
**Pull request by makoteq**
**fix for** https://github.com/fossasia/pslab-android/issues/2035

**Changes**: I added button in navbar with link to google play
**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[app-fdroid-debug.zip](https://github.com/fossasia/pslab-android/files/3988491/app-fdroid-debug.zip)

### CODE REVIEW

**Comment by makoteq** [comment]
@mariobehling Can you review,  please

**Comment by makoteq** [comment]
@CloudyPadmal   Can you review, please

**Comment by cweitat** [comment]
please show a screenshot of how it looks like

**Comment by makoteq** [comment]
GIF-200105_190325
@cweitat

**Comment by cweitat** [comment]
Nice. Looks good. Can the Rate app be above share app and also capital A for the app word?
Icon should be a star instead of message icon as well

**Comment by makoteq** [comment]
[app-fdroid-debug.zip](https://github.com/fossasia/pslab-android/files/4024064/app-fdroid-debug.zip)
I changed icon and move rate button above share button @cweitat

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1037,5 +1037,6 @@
     <string name="no_location_specified">Location not specified</string>
     <string name="compass_default_0">0</string>
     <string name="unknown">Unknown</string>
+    <string name="rateapp">Rate app</string>
```
Comment (cweitat) [inline app/src/main/res/values/strings.xml]: Let's be consistent here. capital A for the app. same as Share App

**Comment by makoteq** [comment]
@cweitat Done

**Comment by makoteq** [comment]
@mariobehling Can you review please?

**Comment by makoteq** [comment]
@mariobehling Can you review please?

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 2]
```diff
@@ -484,21 +490,21 @@ public void onReceive(Context context, Intent intent) {
                             initialisationDialog.dismiss();
                             invalidateOptionsMenu();
                             CustomSnackBar.showSnackBar(findViewById(android.R.id.content),
-                                    getString(R.string.device_connected_successfully),null,null, Snackbar.LENGTH_SHORT);
+                                    getString(R.string.device_connected_successfully), null, null, Snackbar.LENGTH_SHORT);
```
Comment (fcartegnie) [inline app/src/main/java/io/pslab/activity/MainActivity.java]: this is not related to the commit

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 3]
```diff
@@ -484,21 +490,21 @@ public void onReceive(Context context, Intent intent) {
...
                             if (navItemIndex == 0) {
                                 getSupportFragmentManager().beginTransaction().replace(R.id.frame, InstrumentsFragment.newInstance()).commit();
                             } else if (navItemIndex == 1) {
                                 getSupportFragmentManager().beginTransaction().replace(R.id.frame, HomeFragment.newInstance(true, true)).commitAllowingStateLoss();
                             } else {
                                 CustomSnackBar.showSnackBar(findViewById(android.R.id.content),
-                                        getString(R.string.device_connected_successfully),null,null, Snackbar.LENGTH_SHORT);
+                                        getString(R.string.device_connected_successfully), null, null, Snackbar.LENGTH_SHORT);
```
Comment (fcartegnie) [inline app/src/main/java/io/pslab/activity/MainActivity.java]: ditto

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 4]
```diff
@@ -484,21 +490,21 @@ public void onReceive(Context context, Intent intent) {
...
                             }
                         }
                     } else {
                         initialisationDialog.dismiss();
                         Log.d(TAG, "permission denied for device " + device);
                         CustomSnackBar.showSnackBar(findViewById(android.R.id.content),
-                                getString(R.string.device_not_found),null,null, Snackbar.LENGTH_SHORT);
+                                getString(R.string.device_not_found), null, null, Snackbar.LENGTH_SHORT);
```
Comment (fcartegnie) [inline app/src/main/java/io/pslab/activity/MainActivity.java]: ditto

**Comment by makoteq** [comment]
@fcartegnie Done!

**Comment by fcartegnie** [comment]
No. You made it worse.
You must not include unrelated changes.

**Comment by makoteq** [comment]
@fcartegnie I dont really understand, how i can do this?

**Comment by makoteq** [comment]
@fcartegnie Now these changes are not include

**Comment by makoteq** [comment]
@fcartegnie  Now these changes are not include

**Comment by fcartegnie** [comment]
> @fcartegnie I dont really understand, how i can do this?

Learn to squash/fixup commits.
This is incremental changes, not text editor. You can't have do and undo commits.

File: app/src/main/res/values-ru/strings.xml [hunk 5]
```diff
@@ -1,21 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">PSLab</string>
-    <string name="openDrawer">открыть</string>
-    <string name="closeDrawer">закрыть</string>
+    <string name="openDrawer">open_drawer</string>
```
Comment (fcartegnie) [inline app/src/main/res/values-ru/strings.xml]: why did you revert the translations ?

**Comment by makoteq** [comment]
@fcartegnie Sorry for that complications, what do you think now?

**Comment by makoteq** [comment]
@fcartegnie ?

**Comment by makoteq** [comment]
@fcartegnie What about now?

**Comment by fcartegnie** [comment]
I can't approve try & fix commits. Make things clean first

**Comment by makoteq** [comment]
@fcartegnie What do you mean "fix commits" ?

**Comment by makoteq** [comment]
@fcartegnie Now there is only two commits, i think it's quite clean...

**Comment by makoteq** [comment]
@fcartegnie I squashed my commits, i think its ready to merge, also sorry for problems
